### ISSUE TITLE
iPAS AI 應用規劃師科目3練習 - 2026-09-09

### ISSUE THREAD

**Issue by JohnnyLai92** [JohnnyLai92/ipas#19]
# iPAS AI 應用規劃師科目3練習 - 2026-09-09（第 9 輪）

> 側重領域：L231 機器學習基礎數學（機率／統計、線性代數、數值優化）；次要：L232 機器學習與深度學習。
> 情境背景：電商與零售、金融與風控。

## 練習說明

- 本次共 **10 題**：5 題單選、2 題多選、2 題「找出錯誤做法」的反向題、1 題程式／輸出判讀。
- 作答方式：**直接點選每個選項前的核取方塊**即可勾選，不需留言、不需進入編輯畫面。
- 單選題請只勾 1 項；多選題會標示「可勾選 N 項」，請勾滿指定項數；反向題請找出「錯誤／不恰當」的做法。
- 全部作答完成後，請將本 issue 的 label 改為 `ai-analysis`，系統即會進行批改與錯題分析。
- 本練習**不揭露正確答案**，作答完成再由批改流程給出解析。

## 第 1 題〔單選〕L23101 機率／統計

某電商平台分析單日訂單金額，發現大多數訂單集中在 300～800 元，但當天有少數幾筆「企業大宗採購」訂單金額高達數十萬元，使平均值被大幅拉高。若要用**單一數值**代表「一般顧客的典型訂單金額」，下列哪一項統計量最不易受到這些極端大額訂單影響、最為合適？

- [ ] (A) 平均數（Mean）
- [ ] (B) 中位數（Median）
- [ ] (C) 全距（Range）
- [ ] (D) 標準差（Standard Deviation）

---

## 第 2 題〔單選〕L23101 機率／統計（條件機率）

某銀行對 1,000 筆信用卡申請進行統計，交叉分類如下：

| | 事後違約 | 未違約 | 合計 |
|---|---:|---:|---:|
| 高負債比 | 60 | 140 | 200 |
| 低負債比 | 40 | 760 | 800 |
| 合計 | 100 | 900 | 1,000 |

若已知某位申請人屬於「高負債比」族群，則其**事後違約**的條件機率 P(違約 | 高負債比) 最接近下列何者？

- [ ] (A) 6%
- [ ] (B) 20%
- [ ] (C) 30%
- [ ] (D) 60%

---

## 第 3 題〔單選 · 程式判讀〕L23102 線性代數（矩陣維度）

某電商推薦系統以下列程式計算一批交易的線性層輸出，`X` 為「128 筆交易 × 10 個特徵」的資料矩陣：

```python
import numpy as np

X = np.random.rand(128, 10)   # 128 筆交易, 每筆 10 個特徵
W = np.random.rand(10, 4)     # 權重矩陣
b = np.random.rand(4)         # 偏差項
Z = X @ W + b
print(Z.shape)
```

執行後 `print(Z.shape)` 的輸出為何？

- [ ] (A) (128, 4)
- [ ] (B) (10, 4)
- [ ] (C) (128, 10)
- [ ] (D) (4, 128)

---

## 第 4 題〔單選〕L23103 數值優化（一步梯度下降）

某風控評分模型以梯度下降更新單一參數 w。已知目前 w = 1.0，學習率 η = 0.2，此步計算得到的梯度 ∂L/∂w = 3。依參數更新公式 w_new = w − η × (∂L/∂w)，更新後的 w_new 為何？

- [ ] (A) 0.4
- [ ] (B) 0.6
- [ ] (C) 1.6
- [ ] (D) −0.6

---

## 第 5 題〔找出錯誤做法 · 單選〕L23103 數值優化

某電商團隊在調校深度模型的梯度下降訓練時提出下列做法，其中**哪一項是錯誤／不恰當**的？

- [ ] (A) 觀察到 Loss 曲線劇烈震盪甚至發散時，優先嘗試調降學習率
- [ ] (B) 為了讓模型盡快收斂，把學習率設得越大越好，數值越大越安全
- [ ] (C) 學習率設得過低時，訓練會非常緩慢、需要更多迭代才收斂
- [ ] (D) 可搭配學習率衰減（learning rate decay），在訓練後期逐步縮小學習率以穩定收斂

---

## 第 6 題〔單選〕L23201 機器學習原理

某銀行建立信用違約預測模型，訓練資料上的 AUC 高達 0.96，但在從未見過的測試資料上 AUC 僅 0.67。假設資料切分正確、無資料外洩，下列何者最能描述此現象？

- [ ] (A) 模型欠擬合（Underfitting），偏差（Bias）過高
- [ ] (B) 模型過擬合（Overfitting），變異（Variance）過高、泛化能力不足
- [ ] (C) 模型已達最佳泛化，兩者差距屬正常現象
- [ ] (D) 測試集標籤有誤，與模型能力無關

---

## 第 7 題〔多選（可勾選 3 項）〕L23202 常見機器學習演算法

關於常見機器學習演算法的特性，下列敘述**正確**的有哪些？（請勾選 3 項）

- [ ] (A) K-Means 屬於非監督式分群，使用前通常需事先指定群數 K
- [ ] (B) 決策樹能處理非線性關係，且可用 if-then 規則呈現，具較佳可解釋性
- [ ] (C) 線性迴歸（Linear Regression）最適合直接用於二元分類並輸出類別機率
- [ ] (D) 隨機森林為集成（Ensemble）方法，通常比單一決策樹更不易過擬合
- [ ] (E) KNN 在訓練階段即學出明確的參數方程式，推論時不需保留訓練樣本

---

## 第 8 題〔找出錯誤做法 · 單選〕L23203 深度學習原理與框架

某金融科技團隊訓練一個「將交易分為 5 種風險等級」的神經網路分類模型，下列做法中**哪一項是錯誤**的？

- [ ] (A) 輸出層使用 ReLU 激活函數，即可直接得到 5 個類別的機率分布
- [ ] (B) 多分類輸出層應使用 Softmax，使各類別輸出為總和為 1 的機率分布
- [ ] (C) 為緩解過擬合，可在隱藏層加入 Dropout
- [ ] (D) 訓練時若驗證損失不再下降，可用 Early Stopping 提前停止訓練

---

## 第 9 題〔多選（可勾選 3 項）〕L23301 數據準備與特徵工程

某電商建立「顧客流失預測」模型，在資料前處理階段，下列哪些是**正確**的做法？（請勾選 3 項）

- [ ] (A) 先切分 Train/Test，再僅用訓練集計算 StandardScaler 的平均值與標準差，並套用到測試集
- [ ] (B) 對「會員等級（銅／銀／金／白金）」這類有序類別，可使用 Ordinal／Label Encoding 保留順序
- [ ] (C) 為省事，直接用「含測試集」的整體平均值填補所有缺失值
- [ ] (D) 當「流失」樣本極少導致類別不平衡時，可考慮重採樣（如 SMOTE）或調整類別權重
- [ ] (E) 為提高準確率，將測試集資料也一併納入訓練

---

## 第 10 題〔單選〕L23303 模型評估指標

某銀行的詐欺交易偵測模型，業務上最在意的是「**盡量不要漏掉真正的詐欺交易**」（寧可多攔幾筆正常交易人工複查，也不能放過真詐欺）。在此目標下，應優先關注並提升下列哪一項評估指標？

- [ ] (A) Precision（精確率）
- [ ] (B) Recall（召回率）
- [ ] (C) Accuracy（準確率）
- [ ] (D) Specificity（特異度）

---

作答完成後，請記得把 label 改為 `ai-analysis`，即可進行批改與錯題分析。祝作答順利！💪
